### ISSUE TITLE
Should response.writer.write_content implicitly write status, headers?

### ISSUE THREAD

**Issue by Ms2ger** [web-platform-tests/wpt#11764]
Right now, a handler like
```python
         @wptserve.handlers.handler
         def handler(request, response):
             response.writer.write_content("PASS")
```

will write the exact four bytes for "PASS", no status, no headers. The quoted handler came from a unit test that's broken in python3 because its urllib is stricter about responses like that.

**Comment by Hexcles** [comment]
From an API design perspective (and my experience of using various HTTP libraries/web frameworks), I'd say it should, and we could have a separate `write_raw_content` API to write raw responses, but I'm not familiar with the use case of this particular API.

**Comment by jgraham** [comment]
Yes, this is just a bug, I think. Maybe @dhdavvie will get a chance to have a look?

**Comment by dhdavvie** [comment]
Sure I can take a crack at it. @Ms2ger so are you saying the expected functionality should be that if you call `write_content` before having sent headers/status, it should send some default headers and status?

I'm slightly worried about changing the nature of `write_content` as other tests/handlers may already be using it in the way that @Hexcles described, but will do some digging.

**Comment by dhdavvie** [comment]
So it seems like you can just add a check for `content_written`, like so:

    def write_content(self, data):
        """Write the body of the response."""
        if not self.content_written:
            self.write_status(200)
            self.write_default_headers()
            self.end_headers()
        
        #continue

I'll setup this fix in a branch and write some tests for it
